src/client/pps/pps.proto

```diff
@@ -190,14 +203,14 @@ message PipelineInfo {
   bool stopped = 13;
   string outputBranch = 16;
   google.protobuf.Duration scaleDownThreshold = 18;
+  ResourceSpec resources = 19;
```

Comment:
Kinda minor but I would have expected this to be `resource_spec` since the field of type `ParallelismSpec` is called `parallelism_spec` how do you feel about that? I also am not totally sure having users type `spec` everywhere is especially good, but consistency seems more important in this case.